### PR TITLE
Monitor Control 4.1.0

### PR DESCRIPTION
**Pull request by cbleslie**
Hi, I'm new; but look-forward to adding MacOS packages in the future.

###### Description of changes
What's written on the package:
> [MonitorControl](https://github.com/MonitorControl/MonitorControl) - for Apple Silicon and Intel
> Controls your external display brightness and volume and shows native OSD. Use menulet sliders or the keyboard, including native Apple keys!

###### Things done

Regarding Testing: This is an `.app`, no compile; just a simple copy using `undmg`. If tests are needed; let me know what you would like to see beyond a check to make sure the `.app` folder is present.

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [X] x86_64-darwin
  - [X] aarch64-darwin
- [X] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/monitorcontrol/default.nix [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  sourceRoot = "MonitorControl.app";
+
+  installPhase = ''
+    mkdir -p "$out/Applications/MonitorControl.app"
+    cp -R . "$out/Applications/MonitorControl.app"
+  '';
+
+  meta = with lib; {
+    description =
+      "Controls your external display brightness and volume and shows native OSD. Use menulet sliders or the keyboard, including native Apple keys!";
```
Comment (totoroot) [inline pkgs/applications/misc/monitorcontrol/default.nix:27]: See the nixpkgs manual for instructions for writing package descriptions:
https://nixos.org/manual/nixpkgs/stable/#var-meta-description

```suggestion
    description = "A macOS system extension to control brightness and volume of external displays with native OSD";
    longDescription = "Controls your external display brightness and volume and shows native OSD. Use menulet sliders or the keyboard, including native Apple keys!";
```

**Review by totoroot — CHANGES_REQUESTED** [review]
Congrats on your first nixpkgs contribution! 🎉 

Package builds well on my x86_64-darwin machine. Please squash the commits, add yourself as a maintainer in a separate commit and look at my suggestion regarding the package's description.

**Comment by totoroot** [comment]
Result of `nixpkgs-review pr 208042` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>monitorcontrol</li>
  </ul>
</details>

**Comment by cbleslie** [comment]
@totoroot , 
Happy to make changes& thank you for your input. 

Just to be super clear, you want me to:
- Squash the commits
- Then create a new commit on that squash; adding myself, again, to the maintainer list, here:`maintainers/maintainer-list.nix`.
- This would then result in 2 commits in the PR:
  - The package
  - The Maintainer's list update

Is the above correct? Or am I misunderstanding?

Thank you for your patience with me.

**Comment by cbleslie** [comment]
If above is correct: 

- cccba1699af7e74e939e47f0796f1e11c1aa18ed - maintainer list update
- b282c89295c437dad3a95cdb4c5014667247d770 - package addition

Hopefully this is what is needed.

**Comment by cbleslie** [comment]
Updated. :) Now with only commits.

**Review by totoroot — APPROVED** [review]

**Comment by totoroot** [comment]
Result of `nixpkgs-review pr 208042` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>monitorcontrol</li>
  </ul>
</details>

**Pull request merged**